core/presentation/designsystem/src/main/java/com/pronoidsoftware/core/presentation/designsystem/LocalClock.kt

```diff
@@ -0,0 +1,8 @@
+package com.pronoidsoftware.core.presentation.designsystem
+
+import androidx.compose.runtime.staticCompositionLocalOf
+import kotlinx.datetime.Clock
+
+val LocalClock = staticCompositionLocalOf<Clock> {
+    error("No clock provided.")
```

Comment:
So here instead of displaying an error, you could just put `Clock.System` here

Reply:
hm, I think in this instance I might prefer to fail loudly so that I know that I didn't set the clock properly. In MainActivity, I'm setting the composition local provider to provide either the real clock or a test clock. In each other scenario where I'd use this rule, I want this to fail if I've forgotten to set the test clock properly. Considering that I can't use the real clock for screenshot testing because of the comment above 🙂

Reply:
Yes, with the screenshot issue that also makes sense 😄